Fastpack/Error.ml

```diff
@@ -1,11 +1,47 @@
 module Loc = FlowParser.Loc
 module Scope = FastpackUtil.Scope
 
+type color = Cyan | Red | Black | White;;
+
+let print_with_color str col = 
+  let col = match col with 
+    | Cyan -> 36 
+    | Red -> 31
+    | Black -> 30
+    | White -> 37
+  in
+  "\\033[0;" ^ (string_of_int col) ^ "m" ^ str ^ "\\033[0m";;
+
+let get_codeframe (loc:Loc.t) lines = 
+  let startLine = max 0 (loc.start.line - 3) in
+  let endLine = min (List.length lines) (loc._end.line + 1) in
+  let codeframe = (List.filter_map (fun (i,line) -> 
+      if startLine <= i  && i <= endLine then 
+        Some (i, line) 
+      else None
+    ) lines) 
+  in
+  let formatted = List.map (fun (i, line) -> (
+        if loc.start.line <= i && i <= loc._end.line then 
+          let error_substring = String.sub line (loc.start.column) (loc._end.column - loc.start.column) in
```

Comment:
You may want to use the `FastpackUtil.UTF8.sub` here

Reply:
done